src/lib/pages/backoffice/Home/lists/IdleLoansList.js

```diff
@@ -1,31 +1,61 @@
 import { dateFormatter, toShortDate } from '@api/date';
 import { loanApi } from '@api/loans';
+import { withCancel } from '@api/utils';
 import { SeeAllButton } from '@components/backoffice/buttons/SeeAllButton';
 import { Error } from '@components/Error';
 import { Loader } from '@components/Loader';
 import { ResultsTable } from '@components/ResultsTable/ResultsTable';
 import { invenioConfig } from '@config';
+import LiteratureTitle from '@modules/Literature/LiteratureTitle';
 import { BackOfficeRoutes } from '@routes/urls';
 import { DateTime } from 'luxon';
 import PropTypes from 'prop-types';
 import React, { Component } from 'react';
 import { Link } from 'react-router-dom';
 import { Button } from 'semantic-ui-react';
 
-export default class IdleLoansList extends Component {
+export class IdleLoansList extends Component {
+  constructor(props) {
+    super(props);
+
+    this.query = loanApi
+      .query()
+      .withState(invenioConfig.CIRCULATION.loanRequestStates)
+      .withUpdated({
```

Comment:
@kprzerwa I am not sure about this, I would like you to review it. I copied from the existing, did not change anything.
The panel here is for `Loan requests pending since more than 10 days`: shouldn't we check `request_start_date` less than 10 days instead of the updated?

Reply:
it was checking `updated` on purpose, to see if there are some idle requests (as in overlooked and not changed/processed by librarians)

Reply:
I am not sure I understand :( It could be they are updated for other reasons, I am not sure the updated date is reliable.
Why not using the request_start_date?

Reply:
indeed it could be used instead. After some thinking, I remember that we added the `request_start_date` later than we created the dashboard so we used initially `updated` for this feature